### DISCUSSION TITLE
커스텀 월드 개발 시, 제페토 아바타 캐릭터도 원하는 동작을 추가할 수 있나요?

### DISCUSSION THREAD

**Discussion by ellejou76** [naverz/zepeto-studio-kor#687] [5. Q&A]
안녕하세요.

-  커스텀 월드를 기획 개발 중 인데,  제페토 아바타 캐릭터들도 원하는 동작을 하거나 새로운 동작을 추가 할 수 있나요?
   ( V2 정해진 동작이외에는 할 수 없나요? ) 

-  커스텀 동작을 하게 된다면,  동시에 다른 방문자( 동시 접속자) 들에게도 보이게 하는 것이 가능할까요?

감사합니다.

**Comment by smilejsu82** [comment]
네 가능 합니다 ~^^ 

#538
#645 를 참고하세요~

**Reply by ellejou76** [reply]
답변 감사합니다.
그러면, 맵 안에서만 자체적으로 제작한 의상을 입는 것도 가능할까요?
상점에서 판매하는 것이 아닌, 자체의상으로 입기 입니다.

**Reply by smilejsu82** [reply]
> 답변 감사합니다.
> 
> 그러면, 맵 안에서만 자체적으로 제작한 의상을 입는 것도 가능할까요?
> 
> 상점에서 판매하는 것이 아닌, 자체의상으로 입기 입니다.

#609 참고해보세요 ~ 
가능한듯 합니다 ~ 
저는 구매한 에셋 (날개)를 붙여봤는데요 잘 동작 합니다 ~

**Reply by ellejou76** [reply]
> 안녕하세요. smilejsu82 님, 저희가 진행 중인 프로젝트에 협업 문의를 드리고 싶은데, 정보 전달이 가능한 smilejsu82 님의 이메일 주소를 알 수 있을까요? 감사합니다. 제 이메일 주소는 [ellejou76@gmail.com](mailto:ellejou76@gmail.com) 입니다.



> > 답변 감사합니다.
> > 그러면, 맵 안에서만 자체적으로 제작한 의상을 입는 것도 가능할까요?
> > 상점에서 판매하는 것이 아닌, 자체의상으로 입기 입니다.
> 
> #609 참고해보세요 ~ 가능한듯 합니다 ~ 저는 구매한 에셋 (날개)를 붙여봤는데요 잘 동작 합니다 ~

안녕하세요. smilejsu82 님, 저희가 진행 중인 프로젝트에 협업 문의를 드리고 싶은데, 정보 전달이 가능한 smilejsu82 님의 이메일 주소를 알 수 있을까요? 감사합니다. 제 이메일 주소는 ellejou76@gmail.com 입니다.

**Reply by smilejsu82** [reply]
> > 안녕하세요. smilejsu82 님, 저희가 진행 중인 프로젝트에 협업 문의를 드리고 싶은데, 정보 전달이 가능한 smilejsu82 님의 이메일 주소를 알 수 있을까요? 감사합니다. 제 이메일 주소는 [ellejou76@gmail.com](mailto:ellejou76@gmail.com) 입니다.
> 
> 
> 
> 
> 
> 
> 
> > > 답변 감사합니다.
> 
> > > 그러면, 맵 안에서만 자체적으로 제작한 의상을 입는 것도 가능할까요?
> 
> > > 상점에서 판매하는 것이 아닌, 자체의상으로 입기 입니다.
> 
> > 
> 
> > #609 참고해보세요 ~ 가능한듯 합니다 ~ 저는 구매한 에셋 (날개)를 붙여봤는데요 잘 동작 합니다 ~
> 
> 
> 
> 안녕하세요. smilejsu82 님, 저희가 진행 중인 프로젝트에 협업 문의를 드리고 싶은데, 정보 전달이 가능한 smilejsu82 님의 이메일 주소를 알 수 있을까요? 감사합니다. 제 이메일 주소는 ellejou76@gmail.com 입니다.

메일 드렸습니다 ^^
